src/components/Pages/Auth/Registration/FormCompletion.js

```diff
@@ -68,17 +68,13 @@ export default function FormCompletion({
   }
 
   const suggestUsername = async () => {
-    let suggestion = firstName.charAt(0).toUpperCase() + firstName.substring(1) + lastName.charAt(0).toUpperCase() + lastName.substring(1);
-    let number = 0;
-    
-    while (! await validateUserName(suggestion)) {
-        suggestion = firstName.charAt(0).toUpperCase() + firstName.substring(1) + lastName.charAt(0).toUpperCase() + lastName.substring(1) + "-" + number;
-        number += 1;
-    }
+    const template = firstName.charAt(0).toUpperCase() + firstName.substring(1) + lastName.charAt(0).toUpperCase() + lastName.substring(1);
```

Comment:
Let's make the template end with the first letter of the last name, not the whole  last name